src/test/kotlin/com/shortener/service/UrlServiceTest.kt

```diff
@@ -0,0 +1,175 @@
+package com.shortener.service
+
+import com.shortener.domain.UrlEntry
+import com.shortener.domain.UrlEntryRepository
+import com.shortener.dto.UrlShortenRequest
+import com.shortener.exception.InvalidInputUrl
+import com.shortener.utils.IdEncoder
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.Mockito
+import org.mockito.junit.jupiter.MockitoExtension
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.mock.mockito.SpyBean
+import java.time.LocalDateTime
+import javax.transaction.Transactional
+
+
+@ExtendWith(MockitoExtension::class)
+@SpringBootTest
+@Transactional
+class UrlServiceTest(
```

Comment:
https://mockk.io/ I like this for kotlin more than mockito

Reply:
Will use it next time, as for now I've excluded mockito from the tests, don't need it anymore